src/tseda/vpages/map.py

```diff
@@ -56,24 +55,53 @@ def __panel__(self):
         )
         color = color.loc[~gdf.geometry.is_empty.values]
         gdf = gdf[~gdf.geometry.is_empty]
-        return gdf.hvplot.points(
-            hover_cols=["name", "population", "sample_set_id"],
-            geo=True,
-            tiles=self.tiles,
-            tiles_opts={"alpha": 0.5},
-            max_height=self.height,
-            min_height=self.height,
-            size=100,
-            color=color,
-            tools=["wheel_zoom", "box_select", "tap", "pan", "reset"],
-            fill_alpha=0.5,
-            line_color="black",
-            responsive=True,
-        )
+
+        if gdf.empty:
+            gdf = geopandas.GeoDataFrame(
+                pd.DataFrame(index=[0]),
+                geometry=geopandas.points_from_xy([0.0], [0.0]),
+            )
+
+            return gdf.hvplot(  # repeating parameters suboptimal
```

Comment:
Since the parameters to hvplot are identical (except hover_cols) I suggest you put them in a dictionary outside the if clause:

```python
kw = {'geo': True,
          'tiles': self.tiles, 
...}

if gdf.empty:
    kw["hover_cols"] = None
    return gdf.hvplot(**kw)
```

or something similar

Reply:
Ah, thank you for the suggestion. I tried something similar earlier which didn't work but Ive included your comment now and it works!